src/main/java/com/fasterxml/jackson/dataformat/xml/XmlTagProcessors.java

```diff
@@ -0,0 +1,212 @@
+package com.fasterxml.jackson.dataformat.xml;
+
+import java.util.Base64;
+import java.util.regex.Pattern;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+/**
+ * Contains default XML tag name processors.
+ * <p>
+ * Processors should be set in the {@link XmlMapper#setXmlTagProcessor(XmlTagProcessor)}
+ * and/or the {@link XmlMapper.Builder#xmlTagProcessor(XmlTagProcessor)} methods.
+ *
+ * @since 2.14
+ */
+public final class XmlTagProcessors {
+
+    /**
+     * Generates a new tag processor that does nothing and just passes through the
+     * tag names. Using this processor may generate invalid XML.
+     * <p>
+     * With this processor set, a map with the keys {@code "123"} and
+     * {@code "$ I am <fancy>! &;"} will be written as:
+     *
+     * <pre>{@code
+     * <DTO>
+     *     <badMap>
+     *         <$ I am <fancy>! &;>xyz</$ I am <fancy>! &;>
+     *         <123>bar</123>
+     *     </badMap>
+     * </DTO>
+     * }</pre>
+     * <p>
+     * This is the default behavior for backwards compatibility.
+     *
+     * @since 2.14
+     */
+    public static XmlTagProcessor newPassthroughProcessor() {
+        return new PassthroughTagProcessor();
```

Comment:
Not a big deal but if that is immutable (which I think it is), could create and return shared singleton.

Reply:
Yes, it is immutable and it can be done. However, I don't get the reason for such a change. There should barely be any performance (and memory) overhead since processors are only created once for each `ObjectMapper`. Additionally, the `new*Processor` method names would no longer fit, since no new objects are created with a singleton (which is not what is implied with `new*`)

I have no strong opinions for or against this change, just curious.